app/tasks/maintenance/backfill_api_key_owners_task.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class Maintenance::BackfillApiKeyOwnersTask < MaintenanceTasks::Task
+  def collection
+    ApiKey.where(owner_id: nil).or(ApiKey.where(owner_type: nil))
+  end
+
+  def process(api_key)
+    api_key.owner ||= api_key.user
+    api_key.save!(validate: false)
```

Comment:
Could just `api_key.save!`.

Reply:
needs the `validate: false` so we can also backfill on expired/soft deleted API keys

Reply:
They become invalid in those cases?

Reply:
yes, unfortunately